.changeset/thick-tips-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-react-components-development-kitchen': patch
```

Comment:
```suggestion
'@guardian/source-react-components-development-kitchen': minor
```

Is this a minor as it's introducing new non-breaking changes instead of being a bug fix (patch)?